framework-cloud/framework-git/src/main/java/org/kie/cloud/git/AbstractGitProvider.java

```diff
@@ -27,6 +28,13 @@
 
 public abstract class AbstractGitProvider implements GitProvider {
 
+    @Override
+    public String createGitRepositoryWithPrefix(String repositoryPrefixName, String repositoryPath) {
+        String repoName = repositoryPrefixName + "-" + UUID.randomUUID().toString().substring(0, 4);
+        createGitRepository(repoName, ClassLoader.class.getResource(repositoryPath).getFile());
```

Comment:
Why don't you just pass repositoryPath parameter to createGitRepository(...) ? Does it have some reason to retrieve a resource?